packages/engine/lib/orchestrator/src/process/local.rs

```diff
@@ -10,11 +10,7 @@ use hash_engine_lib::{
 
 use crate::process;
 
-#[cfg(debug_assertions)]
-const PROCESS_PATH_DEFAULT: &str = "./target/debug/hash_engine";
-
-#[cfg(not(debug_assertions))]
-const PROCESS_PATH_DEFAULT: &str = "./target/release/hash_engine";
+const PROCESS_PATH_DEFAULT: &str = env!("CARGO_BIN_FILE_HASH_ENGINE");
```

Comment:
Is this equivalent? What happens if you're not running through cargo

Reply:
The artifact is built as a dependency, e.g. `target/debug/deps/artifact/hash_engine-abcdeadbeef`. We could use the old path as fallback.

Reply:
Are we not able to generate a clean `hash_engine` file anymore then?

Reply:
I added a commit to use `target/<debug|release>/hash_engine` if the artifact was not generated.